src/platform/imx8m/include/platform/lib/dma.h

```diff
@@ -15,7 +15,7 @@
 /* max number of supported DMA channels */
 #define PLATFORM_MAX_DMA_CHAN	32
 
-#define DMA_ID_EDMA0	0
+#define DMA_ID_SDMA2	0
 #define DMA_ID_HOST	1
```

Comment:
good catch. I think we can make the following PRs:

1) PR with cleanups in preparation for SDMA driver.
2) PR adding SDMA driver

We could get 1) accepted faster so that i.MX8MP boots fine with qemu.